lib/formats/swagger_1.js

```diff
@@ -15,11 +15,14 @@ var Swagger1 = module.exports = function() {
   Swagger1.super_.apply(this, arguments);
   this.format = 'swagger_1';
 
-  this.converters.swagger_2 = Promise.method(swagger1 => {
+  this.converters.swagger_2 = Promise.method((swagger1, passthrough) => {
     var swagger2 = SwaggerConverter.convert(
       swagger1.spec,
       swagger1.subResources,
-      {buildTagsFromPaths: true}
+      {buildTagsFromPaths: (passthrough && passthrough.buildTagsFromPaths) === undefined
```

Comment:
Instead, lets set:

```js
passthrough = passthrough || {};
passthrough.buildTagsFromPaths = passthrough.buildTagsFromPaths || true;
var swagger2 = SwaggerConverter.convert(
  swagger1.spec,
  swagger1.subResources,
  passthrough
);
```

That way if there's another option someone wants to pass in, they don't have to edit this file.

Reply:
`passthrough.buildTagsFromPaths || true` that won't work if `passthrough.buildTagsFromPaths === false` right?

I went for `passthrough.buildTagsFromPaths !== false`

Reply:
Good catch!